client/src/components/ProfileEdit/index.js

```diff
@@ -194,6 +157,8 @@ const ProfileEdit = function () {
       },
       body: JSON.stringify(cleanForm),
     })
+      .then((res) => res.json())
+      .then(({ profile }) => setProfile(profile))
       .then(() => setIsSaved(true));
```

Comment:
You'll likely have to update the user.context as well. A changed email does not show on the front end until the app is refreshed